build-system/tasks/storybook-react/OWNERS

```diff
@@ -0,0 +1,20 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+        {name: 'ampproject/wg-infra'},
+        {name: 'ampproject/wg-bento'},
+        {name: 'ampproject/wg-components'},
+        {name: 'ampproject/wg-performance'},
+        {name: 'alanorozco', notify: true},
+      ],
+    },
+    {
+      pattern: '**/*.{js,md}',
+      owners: [{name: 'ampproject/working-groups'}],
```

Comment:
Should this apply to the whole directory, rather than the list of groups above (given they overlap significantly)?

Reply:
would it just look like this?

```suggestion
      pattern: '**/*',
```

Reply:
It would be the rule above, with no `pattern` specified at all